packages/web/src/common/store/pages/feed/lineup/sagas.ts

```diff
@@ -51,7 +51,7 @@ function* getTracks({
 
   const params: GetSocialFeedArgs = {
     offset,
-    limit: offset + limit,
+    limit,
```

Comment:
wat.. this seems right but why was it ever offset + limi 🫤

Reply:
it was a hack i think to minimize this issue